specification/batch/data-plane/Microsoft.Batch/stable/2020-03-01.11.0/BatchService.json

```diff
@@ -574,7 +574,7 @@
           "Accounts"
         ],
         "operationId": "Account_ListPoolNodeCounts",
-        "description": "Gets the number of Compute Nodes in each state, grouped by Pool.",
+        "description": "Gets the number of Compute Nodes in each state, grouped by Pool. Note the numbers returned by the operations may not be up to date. If need the accurate up to date count, use a list query to count the nodes resources.",
```

Comment:
If need the accurate up to date count => If the accurate up to date count is needed